std/js/_std/HxOverrides.hx

```diff
@@ -40,7 +40,7 @@ class HxOverrides {
 		switch( s.length ) {
 		case 8: // hh:mm:ss
 			var k = s.split(":");
-			var d = js.Syntax.new_(Date);
+			var d = js.Syntax.construct(Date);
 			(cast d)[cast "setTime"](0);
```

Comment:
I wonder why we do array access here instead of simply `d.setTime`...

Reply:
That's not in the scope of this PR :)